BAZProject/DetailMovieViper/RecommendationViper/Interactor/DetailMovieRecommendationRemoteDataManager.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DetailMovieRecommendationRemoteDataManager.swift
+//  BAZProject
+//
+//  Created by 1050210 on 17/02/23.
+//
+
+import Foundation
+
+class DetailMovieRecommendationRemoteDataManager: DetailMovieRecommendationRemoteDataManagerInputProtocol {
+    var remoteRequestHandler: DetailMovieRecommendationRemoteDataManagerOutputProtocol?
+    
+    private let apiKey : String = "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"
+    private let movieApi : MovieAPI = MovieAPI()
+    
+    func getRecommendation(idMovie: Int?) {
+        guard let idMovie = idMovie else { return }
+        let urlRecommendations = "https://api.themoviedb.org/3/movie/\(idMovie)/recommendations?api_key=\(apiKey)"
```

Comment:
Maybe you can consider to have a URLBuilder class to help you constructing the URLs by using `URLComponents`. In that way you'll avoid having big strings that need to be casted to URLs after